tests/test_schema_validation.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from gordon.schema.validate import JsonValidator
+from gordon import exceptions
+
+import pytest
+
+
+@pytest.fixture
+def validator():
+    return JsonValidator()
+
+
+valid_audit_log = {
+    'methodName': 'v1.compute.instances.insert',
+    'resourceName': '/projects/project-a/zones/zone-b/instances/vm-abc'
+}
+valid_record_a = {
+    'name': 'test1-hostname-a1.subdomain.example.com',
+    'target': '10.0.0.1'
+}
+valid_record_cname = {
+    'name': 'some_domain.example.com',
+    'target': 'some_other_name.somewhere.else.com',
+    'type': 'CNAME',
+    'action': 'create'
+}
+
+
+@pytest.mark.parametrize('valid_json,name', [
+    (valid_audit_log, 'google-audit-log'),
+    (valid_record_a, 'record-action'),
+    (valid_record_cname, 'record-action')
+])
+def test_json_validator_success(validator, valid_json, name):
```

Comment:
Mind adding simple docstrings for test cases, per the [contributors guide](https://github.com/spotify/gordon/blob/master/CONTRIBUTING.rst#code)?

Reply:
Oops! Of course.